### DISCUSSION TITLE
No border for Editor and Pickers in Mac and iOS

### DISCUSSION THREAD

**Discussion by karthikraja-arumugam** [dotnet/maui#11087] [Q&A]
Is there any way to enable around border for Editor, DatePicker, TimePicker and Picker controls as like Entry. While having all these input views in a form, the view is not looking good without borders.

<img width="778" alt="Screenshot 2022-11-03 at 7 56 22 PM" src="https://user-images.githubusercontent.com/92788387/199747302-d9a14c97-d5e1-473e-aa34-11f5900ac51b.png">

**Comment by DonaldConsoleII** [comment]
I'm going to assume this is done through handlers, but haven't worked out the right process for it yet.

**Reply by FM1973** [reply]
@DonaldConsoleII 
Yes, you can use handlers for this.
I did it the other way around and disabled border settings on Android and IOS and I´m using a Border around the controls.
